.github/workflows/integration_test.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y clang cmake curl libgtk-3-dev ninja-build pkg-config unzip xvfb
-          sudo apt install -y dbus-x11 network-manager upower
+          sudo apt install -y dbus-x11 network-manager ubuntu-desktop upower
```

Comment:
i don't know how much it saves but did you give it a try with ubuntu-desktop-minimal?

Reply:
I'll check - if it still takes so long we can just add gnome-shell-extension-desktop-icons-ng